test/unit/requester-core.test.js

```diff
@@ -79,6 +79,55 @@ describe('requester util', function () {
             });
         });
 
+        describe('Should accept URL irrespective of the case', function() {
```

Comment:
This title needs to be changed.
As discussed, these(parent+childs) titles should clearly describe the test suite in logs.